### ISSUE TITLE
NPC won't go to waypoints.

### ISSUE THREAD

**Issue by evamik** [CitizensDev/Citizens#403]
I made an npc, added waypoints to him, but he won't move, did I miss something? Or is it the plugins fault?

**Comment by MrRamziaB** [comment]
I am having the same problem.  however it is happening to my guards as well.  they wont fight or do anything.  hell they wont even look at you as you walk by.

**Comment by evamik** [comment]
Yeah I didn't mentioned that too, happens to me the same, if I make an npc that looks at me, and I make waypoints, then npc doesn't look at me anymore. And if I remove waypoints it still doesn't do anything. Guards won't attack the targets given. Is it because i'm running it on 1.0.0?

**Comment by Chemox** [comment]
Have that problem too :( really sucks though.

**Comment by Sejsel** [comment]
Get the newest dev build, this is fixed!

**Comment by evamik** [comment]
Nice, just tested the waypoints and it works, a bit laggy setting the start, but it's ok ^^

**Comment by Craigdb** [comment]
The only option I can find for a devbuild is on the main website... the citizens.jar in the zip is corrupt (0kb) and I cannot find anywhere else for a devbuild. Any advice would help as I have this waypoint issue as well, and npcs following the player of that name... obviously issues with 1.0

**Comment by Sejsel** [comment]
It used to be on http://ci.citizensnpcs.net/, however it seems to be down right now

**Comment by Craigdb** [comment]
Thanks for the update. I'll keep an eye on that link when it comes up. Really need to fix this as my players tried to make a patrolling guard and it took a bit of effort to repair. I ended up shutting down the server and manually removing waypoint entries from npc-profiles. I don't want to have to do this when our 1.0 server goes live on Monday ;)
